### PR TITLE
Fixed link to Contributor Covenant in CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by garrettheath4**
also added relevant links to Adafruit product pages in README

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -117,11 +117,13 @@ accordingly.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
```
Comment (tannewt) [inline CODE_OF_CONDUCT.md:120]: Good catch! Would you mind updating it here instead: https://github.com/adafruit/cookiecutter-adafruit-circuitpython/blob/master/%7B%7B%20cookiecutter.library_name%20%7D%7D/CODE_OF_CONDUCT.md ? We copy this file into many repos and will need to do a broader automated patch run to fix it everywhere.

Comment (garrettheath4) [reply CODE_OF_CONDUCT.md:120]: Thanks! And sure, can do. 🙂

Comment (garrettheath4) [reply CODE_OF_CONDUCT.md:120]: Done! https://github.com/adafruit/cookiecutter-adafruit-circuitpython/pull/81

**Review by kattni — APPROVED** [review]
Thank you for fixing this!

**Pull request merged**